docs/source/platform/operation-registry.md

```diff
@@ -386,3 +342,126 @@ By clicking on the URL listed in the `Checking for manifest changes at` message,
+
+- `apollo-server` or `apollo-server-<variant>` to `2.6.3`
+- `apollo-server-plugin-operation-registry` to `0.2.0-alpha.1`.
+
+If operations have not been registered since June 6th, the plugin will
+fallback on the old location. With `debug` enabled, the plugin will log which
+manifest is used and the operations added.
+
+#### To target variants/tags other than "current"(default, no tag)
+
+Operations are now registered against a variant/tag and retrieved from the
+manifest specific to a variant/tag. When a variant/tag is set, the operation
+will be validated against the latest schema published under that variant/tag
+and placed in that variant/tag's manifest
+
+To register operations under a specific variant/tag:
+
+- **client**: Upgrade the `apollo` CLI package to 2.13.0 (by default the variant/tag will be set to the value in the [apollo config](https://www.apollographql.com/docs/references/apollo-config/#option-1-use-the-apollo-schema-registry))
+- **server**: Ensure that a schema has been published to the specified graph variant/tag.
+  This can be done by running `apollo service:push --tag <TAG>`
+- **client**: Ensure that operations have been registered to the specified graph variant/tag.
+  This can be done by running `apollo client:push --tag <TAG>` or modifying the service in the `apollo.config.js`
+- **server**: Set the `schemaTag` field of `apollo-server-plugin-operation-registry` to the targeted graph variant/tag
+
+```js
+const server = new ApolloServer({
+  plugins: [
+    require("apollo-server-plugin-operation-registry")({
+      schemaTag: "prod" // highlight-line
+
+      // suggested before enforcing the safelist
+      debug: true,
+      dryRun: true,
+    })
+  ]
+});
+```
+
+> Note: in order to copy the operations from one tag to another, please contact the Apollo team through Intercom or your shared Slack channel
+
+### Change details
+
+#### Manifest storage location
+
+In the previous version of the operation registry, the operation manifest was
+secured by the schema hash, which provided the shared secret between the
+Apollo cloud and apollo-server. The side-effect of this model was coupling
+operation registration and schema uploads, since a manifest needed to be
+created for each schema publish with a new hash. Concretely the GCS url was:
+
+`/(hash of service id)/(schema hash)/manifest.json`
+
+The new model removes this coupling by using the Apollo api key as the shared
+secret. In order to enable independent secret rotation, the api key is used
+to reference a storage secret that then references the manifests
+
+`/(service id)/(hash of api key)/storage-secret.json` <br>
+`/(service id)/(storage secret)/(tag)/manifest.v2.json`
+
+These changes should be transparent, since the new version of the operation
+registry plugin, `0.2.0-alpha.1`, will attempt to use the new scheme and then
+fallback on the old. All apollo cli versions write to the new
+and old manifest locations without additional configuration.
```

Comment:
I think we might want to specify that tag-specific manifests are only created once an operation is pushed to a tag.